controller/src/pid.cpp

```diff
@@ -20,46 +20,41 @@ limitations under the License.
 #include "types.h"
 
 // Define Variables we'll be connecting to
-static double Setpoint, Input, Output;
+static double Setpoint;
+static double Input;
+static double Output;
 
 // Configure the PID
 // Specify the links and initial tuning parameters
 static float Kp = 2, Ki = 8, Kd = 0;
-static PID myPID(&Input, &Output, &Setpoint, Kp, Ki, Kd, DIRECT);
 
-// These constants won't change. They're used to give names to the pins used:
-// Analog output pin that the LED is attached to
-static const int analogOutPin = LED_BUILTIN;
+// DIRECT means that increases in the output should result in increases in the
+// input.  DIRECT as opposed to REVERSE.
+static PID myPID(&Input, &Output, &Setpoint, Kp, Ki, Kd, DIRECT);
 
 enum class pid_fsm_state {
   reset = 0,
   inspire = 1,
   plateau = 2,
   expire = 3,
   expire_dwell = 4,
-
-  count /* Sentinel */
 };
 
 void pid_init() {
-
-  //@TODO: Raw access that needs to be updated to reference the new sensor
-  // module and maybe the upcoming blower module
-  // because currently does not have the benefits of things like calibration,
-  // etc. that the module methods do. I didn't do this because I have no clue
-  // what the desired units are for the control system.
-  // Initialize PID
-  Input = map(Hal.analogRead(DPSENSOR_PIN), 0, 1023, 0, 255);
   Setpoint = BLOWER_LOW;
+  Input = 0;
+  Output = 0;
 
-  // turn the PID on
+  // Turn the PID on.
   myPID.SetMode(AUTOMATIC);
+  Hal.setDigitalPinMode(BLOWERSPD_PIN, PinMode::HAL_OUTPUT);
 }
 
 void pid_execute(const VentParams &params, SensorReadings *readings) {
   uint16_t cyclecounter = 0;
   enum pid_fsm_state state = pid_fsm_state::reset;
 
+  // TODO(jlebar): Get rid of ramp rate; just go to the set pressure asap.
```

Comment:
What is ramp rate? Could you expand this todo with a bit more context for someone else who might fix it?

Reply:
Done.